c++/src/kj/async-win32.c++

```diff
@@ -133,13 +133,44 @@ private:
   Promise<IoResult> promise;
 };
 
+typedef LONG(NTAPI* NtSetInfoFuncPtr)(HANDLE, ULONG_PTR*, void*, ULONG, ULONG);
+
+NtSetInfoFuncPtr getSetInfoFileFunc()
+{
```

Comment:
Can you please make sure this code follows the prevailing style with regard to brace positioning and indentation width?